qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/QuteErrorCode.java

```diff
@@ -34,7 +37,9 @@
 	InvalidMethodParameter("The method `{0}` in the type `{1}` is not applicable for the arguments `{2}`."), //
 	InvalidVirtualMethod(
 			"The virtual method `{0}` in the type `{1}` is not applicable for the base object `{2}` type."), //
-
+	InfixNotationParameterRequired("A parameter for the infix notation method `{0}` is required."), //
+	InvalidMethodInfixNotation("The method `{0}` cannot be used with infix notation, because it has `{1}` parameters."), //

```

Comment:
because it has more than 1 parameter

Reply:
I changed to `because it has not `1` parameter` because 0 parameter is forbidden too.